init.lua

```diff
@@ -6,6 +6,7 @@ wrench = {
 	plus = true,
 	registered_nodes = {},
 	blacklisted_items = {},
+	META_TYPE_IGNORE = 0,
```

Comment:
What is the purpose of this? The only places it's used doesn't seem necessary. (it can be replaced by `nil`)